deploy/charts/bottlerocket-update-operator/values.yaml

```diff
@@ -111,3 +111,7 @@ logging:
     tracing_filter: "info"
   apiserver:
     tracing_filter: "info"
+
+prometheus:
+  serviceMonitor:
```

Comment:
It's probably a good idea to put this under `prometheus.controller`. This PR has made me realize that we expose metrics for `apiserver` but then don't bubble up the endpoint into our helm chart. I opened https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/564 to track that.

Reply:
introduced an additional hierarchy level `controller`